core/src/actors/chain_manager/mod.rs

```diff
@@ -186,6 +190,40 @@ impl ChainManager {
         )
     }
 
+    fn build_block(&self, msg: &BuildBlock) -> Block {
+        // Get all the unspent transactions and calculate the sum of their fees
+        let mut transaction_fees = 0;
+        let transactions: Vec<Transaction> = self
+            .transactions_pool
+            .iter()
+            .map(|t| {
+                // TODO: t.fee()
+                transaction_fees += 1;
+                *t
+            })
+            .collect();
```

Comment:
_Reminder_
This is pending a refactor: take only as many transactions as they can fit in a block.